src/lib/util/handleTripFinish.ts

```diff
@@ -112,15 +112,13 @@ export async function handleTripFinish(
 	if (clueReceived && perkTier > PerkTier.One) components.addComponents(makeDoClueButton(clueReceived));
```

Comment:
couldnt you also wrap this bit into it 👀 removes the additional perk check

keeping all patron features wrapped together

Reply:
Moving those awaits/database calls into an `if` block save several milliseconds while the one you're referring to saves microseconds, roughly 1000x less.

I try to change the minimum amount required for several reasons, like getting it merged quicker, and avoiding code bloat in other cases.